driver-testsuite/tests/Form/SelectTest.php

```diff
@@ -91,6 +91,15 @@ public function testSetValueSingleSelect()
         $this->assertEquals('10', $select->getValue());
     }
 
+    public function testSetValueByDriverSingleSelect()
+    {
+        $session = $this->getSession();
+        $session->visit($this->pathTo('/multiselect_form.html'));
+        $select = $session->getPage()->find('xpath', '//select[@name="select_number"]');
+        $select->setValue('thirty');
```

Comment:
You should also add test (if such test is missing), that `select by option name` behavior works for multi-selects.